tests/run.sh

```diff
@@ -1,8 +1,11 @@
 #!/bin/bash
 set -x -e
 
-# make sure we are using the right ext4 image when testing
-RELEASE_VERSION="master"
+# we need to make sure we use the correct ext4 image for testing
+if [[ -z "$TEST_BRANCH" ]]; then
```

Comment:
Actually this should be:

```
if [[ -z "$BUILDDIR" ]] && [[ -z "$TEST_BRANCH" ]]; then
```

Because if `BUILDDIR` is set, you are getting your artifacts from the build, not from any branch.